web/gds-user-ui/src/components/Section/JoinUs.tsx

```diff
@@ -125,7 +125,7 @@ const JoinUsSection: React.FC = () => {
                   border="2px solid #fff"
                   _hover={{ bg: '#24a9df' }}
                   as="a"
-                  href="https://trisa.io/members/"
+                  href="https://trisa.dev"
```

Comment:
Should we continue to display the info about a member list or temporarily hide it until we're able to show the list again?

Reply:
Let's temporarily hide it.